picoshare/guest_link.go

```diff
@@ -14,7 +14,8 @@ type (
 		ID             GuestLinkID
 		Label          GuestLinkLabel
 		Created        time.Time
-		Expires        ExpirationTime
+		UrlExpires     ExpirationTime
+		FileExpires    FileLifetime
```

Comment:
Oh, I forgot that we already had a `FileLifetime` type. That makes things a little easier.

Can we change the naming so that we're calling the variable `FileLifetime` rather than `FileExpires`? It seems confusing to have `UrlExpires` and `FileExpires` side by side when they represent different things (`UrlExpires` represents a point in time, whereas `FileExpires` is a relative delta).

I'd also like to be consistent with the existing capitalization so that the `t` is lowercase (`fileLifetime` rather than `fileLifeTime`, `file_lifetime` rather than `file_life_time`).